### PR TITLE
add bountive

### PR DESCRIPTION
**Pull request by Hug0x0**

### CODE REVIEW

File: data/bountive.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Bountive",
+  "description": "Bountive is the first decentralized prize savings protocol on Starknet, empowering users with no-risk prize games. Bountive is based on the proven concept known as “Prize Linked Savings Accounts”. This product helps to boost savings. Now, we’re bringing this to the blockchain.",
+  "short_description": "An innovative solution to save & invest your digital assets",
```
Comment (graemeblackwood) [inline data/bountive.json:4]: @Hug0x0 The short description is only shown on the listing pages, so should be really short, please, e.g. "No risk prize savings games"

**Comment by graemeblackwood** [comment]
@Hug0x0 thanks for this. If we could have a shorter "short description" that would be great, please.

Also no smart contracts?

**Comment by Hug0x0** [comment]
Hey @graemeblackwood I have just modified with the addition of a smaller description and the addition of our factory because our raffle often changes address!

**Comment by graemeblackwood** [comment]
Great, thanks!

**Pull request merged**